.github/workflows/chain-deploy-dev.yaml

```
@@ -0,0 +1,91 @@
+jobs:
+  Development:
+    name: Chainwiki-Development
+    runs-on: ubuntu-latest
+    env:
+      AWS_REGION: eu-central-1
+      KUBECTL_VERSION: 1.23.5
+      CLUSTER_NAME: 'swarm-dev'
+      NAMESPACE: 'swarm-dev'
+
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)